annotation-processor/src/main/java/org/hibernate/validator/ap/checks/ConstraintCheckFactory.java

```diff
@@ -197,7 +202,8 @@ public ConstraintCheckFactory(Types typeUtils, Elements elementUtils, Constraint
 						new ConstraintValidatorCheck( constraintHelper, annotationApiHelper ),
 						new AnnotationTypeMemberCheck( annotationApiHelper, typeUtils ),
 						new CrossParameterConstraintCheck( annotationApiHelper, constraintHelper, typeUtils ),
-						new AnnotationMessageCheck( annotationApiHelper, elementUtils )
+						new AnnotationUserMessageCheck( annotationApiHelper ),
+						new AnnotationDefaultMessageCheck( annotationApiHelper, elementUtils )
```

Comment:
How comes the default message check is specified more than once? I'd have expected it only once for `@Constraint` on an annotation type.